brave/brave-core
# perf(wallet): do not wait for sign-all-txs reqs on init

Resolves https://github.com/brave/brave-browser/issues/37254



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Connect a Solana account to the test dapp
- Initiate all "Sign All Transactions" request types
- The panel should open and allow signing/rejecting the pending request(s)
- Repeat the above with a hardware wallet account

<img width="441" alt="Screenshot 2024-04-02 at 2 50 06 PM" src="https://github.com/brave/brave-core/assets/30185185/1a8c067d-379f-42e0-9263-31a799961167">
